testing/testSchools.py

```diff
@@ -106,14 +110,16 @@ def plot_school_assignment(net: sumolib.net.Net, test_name: str, gen_coords: np.
         plt.plot([gen_coords[p, 0], real_coords[assignment[p], 0]],
                  [gen_coords[p, 1], real_coords[assignment[p], 1]], 'k', label="Assign" if p == 0 else "")
 
-    plt.legend()
-    plt.title(f"School placement test for {test_name}")
+    plt.legend(loc='lower left')  # Set to lower left for continuity in paper
+    plt.axis('off')
+    # plt.title(f"School placement test for {test_name}")  # Title disabled for paper
     plt.show()
 
 
-def get_mean_coords(schools: list):
+def get_mean_coords(schools: list) -> List:
     """
-    Takes a list of schools and returns a list of schools with their coordinates averaged out
+    Takes a list of schools and returns a list of schools with their coordinates averaged out.
+     This is probably suboptimal and could be made better with inverse zip.
```

Comment:
Looks like this documentation is wrong. The argument is not a list of schools, it seems to be a list of edge shapes (a list of list of coords). I don't know what you are using this method for, but consider using `position_on_edge` from `utility.py`.